plugins/modules/ndo_mcp_global_policy.py

```diff
@@ -0,0 +1,336 @@
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: fabric_template
+    name: mcp_global_policy_1
+    state: absent
+
+- name: Delete the MCP Global Policy object with UUID
+  cisco.mso.ndo_mcp_global_policy:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: fabric_template
+    uuid: "{{ mcp_global_policy_1.current.uuid }}"
+    state: absent
+"""
+
+RETURN = r"""
+"""
+
+import copy
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.mso.plugins.module_utils.mso import MSOModule, mso_argument_spec
+from ansible_collections.cisco.mso.plugins.module_utils.constants import ENABLED_OR_DISABLED_TO_BOOL_STRING_MAP
+from ansible_collections.cisco.mso.plugins.module_utils.template import MSOTemplate
+
+
+def main():
+    argument_spec = mso_argument_spec()
+    argument_spec.update(
+        template=dict(type="str", required=True, aliases=["fabric_template"]),
+        name=dict(type="str", aliases=["mcp_global_policy"]),
+        uuid=dict(type="str", aliases=["mcp_global_policy_uuid"]),
+        description=dict(type="str"),
+        admin_state=dict(type="str", choices=["enabled", "disabled"]),
+        key=dict(type="str", no_log=False),
+        per_vlan=dict(
+            type="str",
+            aliases=["per_epg", "mcp_pdu_per_vlan"],
+            choices=["enabled", "disabled"],
+        ),
+        loop_detection_factor=dict(type="int", aliases=["loop_factor", "loop_detection_mult_factor"]),
+        port_disable=dict(
+            type="str",
+            aliases=["port_disable_protection"],
+            choices=["enabled", "disabled"],
+        ),
+        initial_delay_time=dict(type="int", aliases=["initial_delay"]),
+        transmission_frequency_sec=dict(type="int", aliases=["tx_freq"]),
+        transmission_frequency_msec=dict(type="int", aliases=["tx_freq_ms"]),
+        state=dict(type="str", default="query", choices=["absent", "query", "present"]),
+    )
+
+    # Enforcing that a user must specify a name or uuid when
+    # adding, updating or removing even though there is only one policy per template.
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["name", "uuid"], True],
+            ["state", "present", ["name", "uuid"], True],
+        ],
+    )
+
+    mso = MSOModule(module)
+
+    name = module.params.get("name")
+    uuid = module.params.get("uuid")
+    state = module.params.get("state")
+    per_vlan = module.params.get("per_vlan")
+    port_disable = module.params.get("port_disable")
+
+    mso_values = {
+        "name": name,
+        "description": module.params.get("description"),
+        "adminState": module.params.get("admin_state"),
+        "key": module.params.get("key"),
+        "enablePduPerVlan": ENABLED_OR_DISABLED_TO_BOOL_STRING_MAP[per_vlan] if per_vlan else None,
```

Comment:
is it possible to leverage the get() function on the dictionary?
```suggestion
        "enablePduPerVlan": ENABLED_OR_DISABLED_TO_BOOL_STRING_MAP.get(per_vlan),
```

Reply:
Done